routemaster/services/deliver.rb

```diff
@@ -40,6 +43,8 @@ def run
           response = _conn.post do |post|
             post.headers['Content-Type'] = 'application/json'
             post.body = data.to_json
+            post.options.timeout = TIMEOUT
+            post.options.open_timeout = CONNECT_TIMEOUT
```

Comment:
Hmm, I'll dig further into how Faraday & make sure these parameters are copied across from the request config - it was my understanding from the documentation this didn't happen, but I'll confirm.